cohere/summarize.py

```diff
@@ -0,0 +1,6 @@
+from cohere.response import CohereObject
+
+
+class Summary(CohereObject):
+    def __init__(self, text: str) -> None:
+        self.text = text
```

Comment:
initially looked a bit odd to me (shouldn't this field be summary?) but summary.summary also doesn't make sense and this is consistent with generate so fine by me